doc_src/cmds/bind.rst

```diff
@@ -230,7 +233,8 @@ The following special input functions are available:
     move the next word to the killring
 
 ``nextd-or-forward-word``
-    if the commandline is empty, then move forward in the directory history, otherwise move one word to the right
+    if the commandline is empty, then move forward in the directory history, otherwise move one word to the right.
+    If at the end of the commandline, accept the current autosuggestion.
```

Comment:
good catch documenting this as well

"accept the current autosuggestion" is not quite right, it's just one word (same as forward-word)
fixed and merged as c0be74c55a6a0a4d7161e1a82ff804ccdf4443d2, thanks!